3/3scale-istio-adapter/Dockerfiles/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest
+
+MAINTAINER Krishna Harsha Voora <krishvoor@in.ibm.com>
+
+
+# Install Prerequisites
+RUN yum install -y git make gcc wget
+
+RUN cd 
+
```

Comment:
I'm confused on this one - this looks like it is just going to run a bash script, not the 3scalie-istio-adapter?  Also, I see you reference installing go in the build script but the dockerfile doesn't indicate that, nor does it looks like it does anything to create this container?

Reply:
@gerrith3 Please refer README.md that talks about how the container and build script need to be used. Hopefully, that will make it clear.